R/loadFromHumann.R

```diff
@@ -0,0 +1,189 @@
+#' Import HUMAnN results to \code{TreeSummarizedExperiment}
```

Comment:
Just checking: are class names also tagged with `\code{}` ?

Reply:
It seems that the we haven't really unified this. --> It would be good to go through the documentation and unify things (also error messages, e.g., if boolean value is required in input check, the error message is the same in all functions)

Reply:
Right. Is this something to unify (opening an issue)?